Sem. 03/Solutions/Task02.cpp

```diff
@@ -0,0 +1,217 @@
+#include <iostream>
+#include <fstream>
+
+const char* FILE_NAME = "jobOffers.bin";
+const char* ERROR_MESSAGE = "Error";
+
+const int MAX_NAME_LENGTH = 25;
+
+struct JobOffer {
+    char name[MAX_NAME_LENGTH];
+    int coworkerCount;
+    int vacancyDays;
+    long long salary;
+};
+
+int stringLength(const char* string) {
+    int length = 0;
+
+    for (int i = 0; string[i] != '\0'; ++i) {
+        length++;
+    }
+
+    return length;
+}
+
+bool areStringsEqual(const char* str1, const char* str2) {
+    int length1 = stringLength(str1);
+    int length2 = stringLength(str2);
+
+    if(length1 != length2) {
+        return false;
+    }
+
+    for(int i = 0; i < length1; ++i) {
+
+        if(str1[i] != str2[i]) {
+            return false;
+        }
```

Comment:
error if length2 < length1

Reply:
If length1 and length2 are different, it would've returned false (line 31).